src/views/Overlay.js

```diff
@@ -67,7 +67,10 @@ class Overlay extends Component {
   };
 
   handleMouseForAnnotation() {
-    return this.state.mode !== 'MOVE' && this.props.model.controlsactive;
+    return (this.state.mode !== 'MOVE'
```

Comment:
add comment - for these three modes, the UI handing is deferred to osd, and osd-annotations will not intercept the mouse clicks events. This enables right click handling by registering a right click handler with osd, this handler provides fast edit/delete capability.